src/Access/AccessControl.cpp

```diff
@@ -608,7 +608,7 @@ AuthResult AccessControl::authenticate(const Credentials & credentials, const Po
     }
     catch (...)
     {
-        tryLogCurrentException(getLogger(), "from: " + address.toString() + ", user: " + credentials.getUserName()  + ": Authentication failed");
```

Comment:
>this PR demotes the log level of unsuccessful authentication attempts from ERROR to DEBUG

Isn't this demoting from ERROR to INFO, though? What's the rationale for this level as oppose to the one in TCPHandler that is demoted to DEBUG?

Reply:
You are right, I'll make this consistent.

(most logs in ClickHouse are at the DEBUG-level, I'll change both places to that)